src/writing-policies/rego/open-policy-agent/04-distribute.md

```diff
@@ -0,0 +1,139 @@
+# Distribute
+
+We have written, built and run our Rego policy. Now it's time to
+distribute the policy.
+
+Policies have to be annotated in order for them to be executed in the
+`policy-server`, the component that executes the policies when running
+in a Kubernetes cluster.
+
+Also, the command we used to execute from the previous section also
+signals a limitation:
```

Comment:
I don't understand what you mean with "signals a limitation"

Reply:
I explain that right after the command: `As you can see, we had to provide the execution mode`.

Reply:
I still find it confusing tbh. I would delete lines 10 - 30, and then adapt the file. I preferred the approach you used to explain this concept when talking about gatekeeper. You could just copy and paste the gatekeeper file and do some minor changes to it